docs/node/tutorial/01-create-application.md

```diff
@@ -14,6 +14,12 @@ In this tutorial you will use Nx to build a server application out of common lib
 yarn create nx-workspace
 ```
 
+or with global installed nx:
```

Comment:
Could you align it with react/01-create-application.md or angular/create-application.md?